CSE/CSE.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.27004.2009
+VisualStudioVersion = 15.0.27004.2008
```

Comment:
Can you update your VS to current version? I would hesitate downgrading this version.

Reply:
15.0.27004.2009 is my current version, yet the one in the current master is 15.0.27004.2008 so I changed it back